components/MapViewScreen/index.tsx

```diff
@@ -51,20 +51,54 @@ export default function MapViewScreen({
     projectSize: { min: 0, max: 0 },
     location: [],
   });
+  const [filteredWithoutSearch, setFilteredWithoutSearch] =
+    useState<Project[]>(projects);
 
-  const handleFilterChange = (filter: FilterType) => {
-    console.log(filter);
+  // show projects based on selected filters
+  const handleFilterButtonClick = () => {
+    const { status, technology, projectSize, location } = selectedFilters;
+    console.log(
```

Comment:
Before you submit your sprint for review + merging, you should remove all testing console log statements because we don't want this on the deployed app.

Reply:
done